<?xml version="1.0"?>

<launch>
    <!--launch gps waypoint-->
	<node ns="/link_waypoint_nav" pkg="link_waypoint_nav" type="gps_waypoint" name="gps_waypoint" output="screen" />
	<param name="/link_waypoint_nav/coordinates_file" value="/waypoint_files/points_outdoor_park.txt" type="string" />

	<!--launch simulation waypoint-->
	<!--node ns="/link_waypoint_nav" pkg="link_waypoint_nav" type="send_waypoint_sim" name="send_waypoint_sim" output="screen" />
	<param name="/link_waypoint_nav/coordinates_file" value="/waypoint_files/collected_points_sim.txt" type="string" /-->
</launch>
